exercises/parallel-letter-frequency/src/lib.rs

```diff
@@ -1 +1,12 @@
+use std::collections::HashMap;
 
+pub fn frequency(input: &[&str], worker_count: i32) -> HashMap<char, u32> {
```

Comment:
One more note: the output should probably be `HashMap<char, usize>` on the basis that a `str` can contain at most `usize` bytes. This won't affect any of the tests here, but it's good practice.